$OpenBSD: patch-src_exiv2_cpp,v 1.1 2019/02/14 09:55:40 jca Exp $

Use a fixed location for locale files, getProcessPath() doesn't work on
OpenBSD.

Index: src/exiv2.cpp
--- src/exiv2.cpp.orig
+++ src/exiv2.cpp
@@ -129,7 +129,7 @@ int main(int argc, char* const argv[])
 {
 #ifdef EXV_ENABLE_NLS
     setlocale(LC_ALL, "");
-    const std::string localeDir = Exiv2::getProcessPath() + EXV_LOCALEDIR;
+    const std::string localeDir = EXV_LOCALEDIR;
     bindtextdomain(EXV_PACKAGE_NAME, localeDir.c_str());
     textdomain(EXV_PACKAGE_NAME);
 #endif
